main.go

```diff
@@ -80,6 +61,14 @@ func gotEvent(event cloudevents.Event) error {
 	var shkeptncontext string
 	_ = event.Context.ExtensionAs("shkeptncontext", &shkeptncontext)
 
+	logger := keptncommon.NewLogger(shkeptncontext, "", utils.ServiceName)
+
+	// Send prometheus alert
+	if event.SpecVersion() == "" {
+		eventhandling.ProcessAndForwardAlertEvent(event.Data(), logger, shkeptncontext)
+		return nil
+	}
```

Comment:
I think this won't work :) I understand the reasoning behind it though, and code-wise it looks nice.

Anyway, here is why this won't work:

Prometheus itself sends the alert to prometheus-service. This alert does **not** send a proper CloudEvent, also it does not send an shkeptncontext, nor a specversion. I think we will never get into `gotEvent(event cloudevents.Event)` in that case.
An example payload is available in prometheus_alert.json in this repo (I will add a helper tool to execute an HTTP request against localhost:8080 with that payload in another PR).

Anyway, long story short, I tried it out using
```
POST http://localhost:8080/
Accept: application/json
Cache-Control: no-cache
Content-Type: application/json

< ./prometheus_alert.json
```

and I get 
```
Response code: 415 (Unsupported Media Type); Time: 287ms; Content length: 0 bytes
```

This is what I expected, we never get into `gotEvent`, because the CloudEvents library is blocking us here.

Reply:
Related PR: https://github.com/keptn-contrib/prometheus-service/pull/271